cmd/waku/flags_rln.go

```diff
@@ -30,7 +31,7 @@ func rlnFlags() []cli.Flag {
 		},
 		&cli.StringFlag{
 			Name:        "rln-relay-content-topic",
-			Value:       "/toy-chat/2/luzhou/proto",
+			Value:       protocol.NewContentTopic("toy-chat", 3, "mingde", "proto").String(),
```

Comment:
Does it make sense to have a default value for content-topic? Rather we should throw an error if content-topic is not mentioned. 
We can always inlcude in docs which content-topic can be used if someone wants to just test.

Reply:
Note that we plan to remove rln pubsub topics and content topics, so that rln is applied in all the subscribed ones if enabled.

See 1. from https://github.com/waku-org/nwaku/issues/1906
and https://discord.com/channels/1110799176264056863/1111540684575477821/1141378004841410640

Reply:
For this PR let's keep it with the hardcoded default content topic to match nwaku's flag: https://github.com/waku-org/nwaku/blob/master/apps/wakunode2/external_config.nim#L165